src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/NotificationService.java

```diff
@@ -150,11 +150,16 @@ public void sendContestOrderNotApprovedEmail(CallbackRequest callbackRequest) {
         sendNotificationEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(callbackRequest), uri);
     }
 
-    public void sendContestedConsentOrderApprovedEmail(CallbackRequest callbackRequest) {
+    public void sendContestedConsentOrderApprovedEmailToApplicantSolicitor(CallbackRequest callbackRequest) {
         URI uri = buildUri(notificationServiceConfiguration.getContestedConsentOrderApproved());
         sendNotificationEmail(notificationRequestMapper.createNotificationRequestForAppSolicitor(callbackRequest), uri);
     }
 
+    public void sendContestedConsentOrderApprovedEmailToRespondentSolicitor(CallbackRequest callbackRequest) {
```

Comment:
if you have a look at the other implementations we've been creating a base method which in this case would be sendContestedConsentOrderApprovedEmail which takes the notification request, the resp and app methods would then just forward the notification request on to the base method